demos/jans-tent/CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+## [0.4.3](https://github.com/christian-hawk/auth-tdd-client/compare/v0.4.2...v0.4.3) (2023-01-13)
```

Comment:
This will be reset if added to the release strategy. We can't possible start from 0.4.3